kotlin/codegen/pom.xml

```diff
@@ -136,6 +147,12 @@
             </goals>
           </execution>
         </executions>
+        <configuration>
+          <args>
+            <!-- So we can declare ourselves as users of KotlinPoetMetadataPreview -->
+            <arg>-Xuse-experimental=kotlin.Experimental</arg>
```

Comment:
Will `-Xuse-experimental=com.squareup.kotlinpoet.KotlinPoetMetadataPreview` work as well?

Reply:
That shoooould work too, I guess we can do that since we only use one preview annotation

Reply:
3206cdd